packages/fluentui/docs/tsconfig.json

```diff
@@ -4,6 +4,9 @@
     "module": "esnext",
     "paths": {
       "@fluentui/*": ["packages/fluentui/*/src/index"],
+      "@uifabric/merge-styles": ["packages/merge-styles/src"],
```

Comment:
is merge-styles needed here?

Reply:
`utilities` has a dependency on `merge-styles`. When v0 is parsing `utilities` we have to include the path map to `merge-styles` as well, otherwise it will complain about `merge-styles` module not being found.